testutil/ibc_testing/specific_setup.go

```diff
@@ -20,6 +20,30 @@ import (
 	appProvider "github.com/cosmos/interchain-security/app/provider"
 )
 
+// TODO: refactor this out test (-v) verbose flag needed to see output!
+func ProviderAppIniterWithLog() (ibctesting.TestingApp, map[string]json.RawMessage) {
+	l := log.TestingLogger()
+	l = log.NewFilter(l,
```

Comment:
What does this line do?

Reply:
The log has a feature where you add filter k/v pairs, and only lines includes those pairs are shown. By doing this, and specifying `module", "x/ibc-provider` it filters out all the sdk logging (during tests only!)